### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by Datcenko**
BAHC3-176

### CODE REVIEW

**Comment by alex-koshara** [comment]
Please, add method to form `post` or `get`

also, check styles for input when it is hovered. It is layered on other buttons

Снимок экрана 2024-12-02 в 2 33 25 PM

image

**Comment by alex-koshara** [comment]
This divider not from form part. If you'll replace the buttons, its also will be move
Снимок экрана 2024-12-02 в 2 38 53 PM

**Comment by alex-koshara** [comment]
Please delete this selector and use `gap` property. 

This selector is hard to understand, and it can be made easier. Using the gap property will help in this. Moreover, we already have flex on the parent

image

**Comment by alex-koshara** [comment]
Please, check the color in field
image
image

**Comment by alex-koshara** [comment]
Please, check the size of input. In my opinion it should be full width

image

**Comment by alex-koshara** [comment]
Without submit button user can't submit this form
image

**Comment by alex-koshara** [comment]
How do you think, is it good idea to give for users opportunity to click in this area?

Снимок экрана 2024-12-02 в 2 52 42 PM

**Comment by alex-koshara** [comment]
Please add some name and value for all inputs
image

**Comment by alex-koshara** [comment]
Please check fonts for radio buttons. It looks different with checkbox
image

**Comment by alex-koshara** [comment]
It's really cool that you use the form tag correctly and in the right places!

You're also using the label tag correctly. At the same time, you know how to link it to the input field through the wrapper for id and for!

The correctly using radio-buttons! And nice to see the correct type of inputs!

File: index.html [hunk 1]
```diff
@@ -41,7 +41,7 @@ <h1 class="logo">
             </nav>
             <div class="tools">
                 <form class="search">
-                    <button type="submit" class="search__btn">
+                    <button type="submit" method="post" class="search__btn">
```
Comment (alex-koshara) [inline index.html:44]: Suggestion
I think for search we can use method `get`

File: blog-article.html [hunk 2]
```diff
@@ -43,7 +43,7 @@ <h1 class="logo">
             </nav>
             <div class="tools">
                 <form class="search">
```
Comment (alex-koshara) [inline blog-article.html:45]: Please add method to all forms
image

File: blog-article.html [hunk 3]
```diff
@@ -43,7 +43,7 @@ <h1 class="logo">
...
-                    <button type="submit" class="search__btn">
+                    <button type="submit" method="post" class="search__btn">
```
Comment (alex-koshara) [inline blog-article.html:46]: The buttons send the form. And that's why we specify a method for the form, not for the buttons

File: catalog.html [hunk 4]
```diff
@@ -92,241 +92,286 @@ <h1 class="page__title">
...
-                                    <span class="form-checkbox__box">
-                                    </span>
-                                    <span class="form-checkbox__text">
-                                        Robot  
-                                    </span>
-                                </p>
-                            </label>
+                            <div>
+                                <label class="form-checkbox">
+                                    <input type="checkbox" name="" class="input-hidden form-checkbox__input" name="clothing" value="combatant">
```
Comment (alex-koshara) [inline catalog.html:97]: You can reduce your classes. For example:
form-checkbox => checkbox
form-radio => radio

File: catalog.html [hunk 5]
```diff
@@ -92,241 +92,286 @@ <h1 class="page__title">
...
+                                        </span>
+                                        <span class="form-checkbox__text">
+                                            Combatant  
+                                        </span>
+                                    </p>
+                                </label>
+                            </div>
+                            <div>
+                                <label class="form-checkbox">
+                                    <input type="checkbox" class="input-hidden form-checkbox__input" name="clothing" value="ninja">
```
Comment (alex-koshara) [inline catalog.html:109]: It's absolutely correct attrs `name` and `value`!

**Review by alex-koshara — APPROVED** [review]
Nice work! 

Please, pay attention with comments. Let's fix them in the next task

File: index.html (hunk 1 above)
Comment (Datcenko) [reply index.html:44]: Done

File: blog-article.html (hunk 2 above)
Comment (Datcenko) [reply blog-article.html:45]: Done

File: blog-article.html (hunk 3 above)
Comment (Datcenko) [reply blog-article.html:46]: Done

File: catalog.html (hunk 4 above)
Comment (Datcenko) [reply catalog.html:97]: Done

**Pull request merged**